src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -38,6 +39,17 @@ struct ExcisionSphere {
   using group = Worldtube;
 };
 
+/*!
+ * \brief The internal expansion order of the worldtube solution.
+ */
+struct ExpansionOrder {
+  using type = size_t;
+  static constexpr Options::String help{
+      "The internal expansion order of the worldtube solution. Currently only "
+      "order 0 is implemented"};
+  static size_t upper_bound() { return 0; }
```

Comment:
Do you have short-term plans to extend this to higher order? Else you can just hardcode `expansion_order = 0` where you need to instead of going through the trouble of option-creating it.

Reply:
Yes, I will also add orders 1 and 2. But to make reviews easier, I will just add the simplest order 0 scheme first and then add the others on top.